Scripts/Multis/Houses/HouseTeleporterTile.cs

```diff
@@ -383,6 +383,11 @@ public bool CheckTravel(Mobile from, Point3D dest, Map destMap)
                 from.SendLocalizedMessage(1019004); // You are not allowed to travel there.
                 return false;
             }
+            else if (Region.Find(dest, destMap).IsPartOf("Abyss") && from is PlayerMobile pm && !pm.AbyssEntry)
```

Comment:
I actually tried this myself when I thought I could fix it.

I could still use the house tele without abyss entry as a player.

I think it has something to do with the HouseRegion overridding the Abyss region. I tried a ton of ways and could not do it inside of HouseTeleporterTile alone.